.github/workflows/osx_testing.yml

```diff
@@ -0,0 +1,185 @@
+name: testing
+
+on: [push, pull_request]
+
+# TODO: Run testing using luarocks installed module.
```

Comment:
luarocks is irrelevant here (that's not a Lua module).

Reply:
I deleted it.